css/style-who-is-tec.css

```diff
@@ -0,0 +1,95 @@
+/* General Style  */
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+p {
+    Font-family : 'Rowdies', cursive;
+    font-style: normal;
+    font-weight: 300;
+    text-align: center;
+    color: rgb(0, 0, 0);
+}
+
+.container {
+    width: 414px;
+    height: 736px;
+    background: #45BADD;
+    border-radius: 30px;
+}
+
+/* Style For Header Section */
+
+.container-header {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.container-header-logo {
+    width: 67px;
+    height: 49px;
+    margin: 25px 70px 0 0;
+}
+
+.container-header-home {
+    width: 48px;
+    height: 48px;
+    margin: 25px 0 0 70px;
+}
+
+/* Style For content Section */
+
+.container-content {
+    width: 414px;
+    height: 439px;
+}
+
+.container-content-introduction {
+    line-height: 22px;
+    margin-top: -45px;
+    font-size: 15px;
+}
+
+/* Style For Speak Sectin */
+
+.container-speak {
+    display: flex;
+    font-size: 18px;
+    justify-content: center;
+    align-items: center;
+    margin: 30px 0;
+}
+
+.container-speak-paragraph {
+    font-size: 18px;
+    color: rgba(0, 0, 0, 0.7);
+}
+
+#volume {
+    margin-left: 12px;
+}
+
+/* Style For Button */
+
+button {
+    width: 244px;
+    height: 75px;
+    text-align: center;
+    border-radius: 51px;
+    box-shadow: inset 0px 3px 14px rgba(255, 52, 130, 0.33);
+    background-color: #82FF34;
+    margin: 0 85px;
+}
+
+.button-paragraph {
+    font-style: normal;
```

Comment:
**As the previous comment :** 

**`normal` is the default value so, you don't need to add it.**